cmd/krew/cmd/system.go

```diff
@@ -0,0 +1,46 @@
+// Copyright © 2019 NAME HERE <EMAIL ADDRESS>
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package cmd
+
+import (
+	"fmt"
+
+	"sigs.k8s.io/krew/pkg/migration"
+
+	"github.com/spf13/cobra"
+)
+
+// systemCmd represents the system command
+var systemCmd = &cobra.Command{
+	Use:   "system receipts-upgrade",
```

Comment:
can't we nest subcommands? this seems a bit hacky. we should let cobra handle non-existing subcmds etc for us.

Reply:
Yes, good idea.

Reply:
don't forget to change the string :P

Reply:
:man_facepalming: